.github/workflows/pull-request.yaml

```
@@ -65,4 +54,5 @@ jobs:
       - uses: actions/setup-node@v4
         with:
           node-version: ${{ vars.NODEJS_VERSION }}
-      - run: make build
+      - run: npm install --immutable
+      - run: npm run build
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)